test/Test/TZBTC.hs

```diff
@@ -371,8 +371,9 @@ test_mint = testGroup "TZBTC contract `mint` test"
           ]
   ]
 
-test_approvableLedger :: IO TestTree
-test_approvableLedger = testSpec "TZBTC contract approvable ledger tests" $
+-- TODO [#129]: enable this once we bring V2 with the recent ManagedLedger
```

Comment:
I guess we'll still want to test V1, right? There are some building blocks exported:
```
  , alEmptyInitBalanceSpec
  , alTransferSpec
  , alTransferSelfSpec
  , alTransferSelfAlwaysSpec
  , alAllowSpec
```

probably we can construct a non-empty spec from them for V1.

Reply:
I can do that

It just seems to me like the probability of test-breaking changes in V1 is negligible comparing to maintaining costs and probability of missing new building blocks added in morley-ledger (not saying that we are going to throw a decent amount of tests away right now).

Reply:
> the probability of test-breaking changes in V1 is negligible

Sounds fair, but rather than breaking something in V1 we may break something in tests 🤷

Anyway, the idea is that these tests should be easy and straightforward to use, so maintenance costs should be pretty small. If we get tired of maintenance, let's remove them.